tests/iotools/test_pvgis.py

```diff
@@ -630,23 +622,6 @@ def test_read_pvgis_tmy_csv(expected, month_year_expected, inputs_expected,
                                meta_expected, csv_meta, pvgis_data)
 
 
-def test_read_pvgis_tmy_basic(expected, meta_expected):
-    fn = TESTS_DATA_DIR / 'tmy_45.000_8.000_2005_2023.txt'
-    # XXX: can't infer outputformat from file extensions for basic
-    with pytest.raises(ValueError, match="pvgis format 'txt' was unknown"):
```

Comment:
there are not tests now for raising an exception if either the file extension or `pvgis_format` was not `'json'` or `'csv'`. Don't need the basic test file, can create a named temp file instead.

Reply:
There is now a dedicated test that checks if an error is raised when a filename with a non-recognized file format is passed.